.github/workflows/integration-tests.yml

```diff
@@ -20,6 +20,11 @@ jobs:
         with:
           python-version: "3.8"
 
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y freetds-dev libkrb5-dev
```

Comment:
relevant?

Reply:
These system dependencies are relevant when `pymssql` needs to be built from source because no pre-built wheel is available for your specific Python version and platform. Without them it's currently broken, so not sure what was changed in the runner.